android/app/src/main/res/layout-land/onmap_downloader.xml

```diff
@@ -45,14 +45,20 @@
         android:gravity="center_horizontal"
         tools:text="Some name very loooooooooong name"
         tools:ignore="UnusedAttribute"/>
+
       <TextView
-        android:id="@+id/downloader_size"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_margin="@dimen/margin_half"
-        android:textAppearance="@style/MwmTextAppearance.Body2"
-        android:gravity="center_horizontal"
-        tools:text="1000 MB"/>
+          android:id="@+id/downloader_size"
+          android:layout_width="match_parent"
+          android:layout_height="24sp"
```

Comment:
Will it work correctly if user sets larger/smaller font size in system settings? Does OM respect these settings?

Reply:
<img width="343" alt="Screenshot 2024-08-15 at 11 18 48 AM" src="https://github.com/user-attachments/assets/8967be24-704a-4e3d-adf0-6ecdd72d5382">

Looks like we honor it but then `Download` button breaks and progress gets cut off (as in original). At the very largest size having a `minTextSize` of `8sp` works on my emulator. How deep down the rabbit hole do you want me to go? I can tweak the `Download` button to work similarly as the progress.